src/vs/server/remoteExtensionHostAgentServer.ts

```diff
@@ -290,11 +293,13 @@ export class RemoteExtensionHostAgentServer extends Disposable {
 
 		let appInsightsAppender: ITelemetryAppender = NullAppender;
 		if (supportsTelemetry(this._productService, this._environmentService)) {
-			if (this._productService.aiConfig && this._productService.aiConfig.asimovKey) {
+			if (this._productService.aiConfig && this._productService.aiConfig.asimovKey !== 'foo') {
```

Comment:
Is this left over, or genuine code? 😆

Reply:
It's intended, I could remove that block of code, but this way I think it would create less merge conflicts